client/landing/gutenboarding/onboarding-block/create-site/style.scss

```diff
@@ -1,83 +1,54 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../mixins';
+@import '../colors';
 
 $progress-step-title-height: 40px;
 $progress-step-number-step-height: 45px;
 $progress-duration: 800ms;
 $progress-widget-height: 300px;
 
-.create-site__background {
-	position: fixed;
-	top: 0;
-	left: 0;
-	right: 0;
-	bottom: 0;
-	z-index: 50;
-	background-color: $white;
+.create-site {
+	padding: 1em;
+	margin: 44px 0 50px;
+	max-width: 540px;
 
-	&.gutenboarding-page {
-		max-width: none;
+	@include break-mobile {
+		margin: 30vh auto;
 	}
 
-	.create-site__header {
-		display: flex;
-		justify-content: space-between;
-		align-items: center;
-		height: 56px;
-		padding: 8px;
-	}
-
-	.create-site__content {
-		position: absolute;
-		top: calc( 50% - #{$progress-widget-height / 4} );
```

Comment:
I think removing this container might be causing the progress bar to no longer be centred vertically on mobile.

On production, the mobile layout of the create-site step looks like:

![image](https://user-images.githubusercontent.com/14988353/84626603-bba61d80-af28-11ea-8126-d2ace3c10319.png)

With this change, it's moved to the top:

![image](https://user-images.githubusercontent.com/14988353/84626754-0aec4e00-af29-11ea-8752-63ba07cf51d2.png)